apps/native-component-list/src/screens/ModulesCore/CoreModuleScreen.tsx

```diff
@@ -12,6 +14,15 @@ export default function ExpoCoreModuleScreen() {
           reloadAppAsync('Reload from ExpoCoreModuleScreen');
         }}
       />
+
+      <HeadingText>expoModulesCoreVersion</HeadingText>
+      <MonoText>{JSON.stringify(globalThis.expo.expoModulesCoreVersion, null, 2)}</MonoText>
```

Comment:
Could you add types for `expo` object by modifying https://github.com/expo/expo/blob/2f27af64180e717c1f87575f18c3b6b2dbfbb1e1/packages/expo-modules-core/src/ts-declarations/global.ts#L6